include/deal.II/matrix_free/mapping_info_storage.h

```diff
@@ -304,6 +293,16 @@ namespace internal
       unsigned int
       quad_index_from_n_q_points(const unsigned int n_q_points) const;
 
+      /**
+       * Helper function to determine which update flags must be set in the
+       * internal functions to initialize all data as requested by the user.
+       */
+      static UpdateFlags
+      compute_update_flags(
+        const UpdateFlags                                     update_flags,
+        const std::vector<dealii::hp::QCollection<spacedim>> &quad =
```

Comment:
The fact that the variable is a vector should be reflected in the name of the variable (i.e. plural form should be used in my opinion). There are other occurrences that I do not comment.